.pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-step-template.yaml

```diff
@@ -74,5 +74,44 @@ steps:
       sudo go test -count=1 datapath_linux_test.go -timeout 1m -tags connection -run ^TestDatapathLinux$ -tags=connection,integration -isDualStack=true
       echo "Dualstack Overlay Linux datapath IPv4 test"
       sudo go test -count=1 datapath_linux_test.go -timeout 1m -tags connection -run ^TestDatapathLinux$ -tags=connection,integration
+      echo "removing load-test namespace"
+      kubectl delete ns load-test
     name: "DualStack_Overlay_Linux_Tests"
-    displayName: "DualStack Overlay Linux Tests"
\ No newline at end of file
+    displayName: "DualStack Overlay Linux Tests"
+
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        make -C ./hack/aks windows-nodepool-up AZCLI=az VM_SIZE=$(DUALSTACK_VM_SIZE) SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) CLUSTER=${{ parameters.clusterName }}-$(make revision)
+        echo "Windows node are successfully added to Dualstack Overlay Cluster"
+        kubectl cluster-info
+        kubectl get node
+        kubectl get po -owide -A
+        echo "Deploying windows CNS"
+        kubectl apply -f test/integration/manifests/cns/daemonset-windows.yaml
```

Comment:
Shouldn't we use the latest CNS generated from this pipeline or is this temporary to ensure that you have a stable CNS as you test?